setup.py

```diff
@@ -109,7 +109,7 @@ def check_nonpython_dependencies():
     extras_require={
         'docs': DOC_REQS,
         'test': TEST_REQS,
-        'dev': ['black', 'flake8'] + DOC_REQS + TEST_REQS + DEPLOY_REQS,
+        'dev': ['black==20.8b1', 'flake8'] + DOC_REQS + TEST_REQS + DEPLOY_REQS,
```

Comment:
any particular reason why black is locked to 20.8b1?

Reply:
I just locked it to the latest version for now so that we are consistent